docs/docs/porting-an-html-site-to-gatsby.md

```diff
@@ -0,0 +1,543 @@
+    <div className="bio-card">
+      <img
+        alt="Comically crude stick person sketch"
+        src="/person.png"
+      />
+      <p>Ella</p>
+    </div>
+  </Layout>
+)
+```
+
+The other 2 `Who We Are` pages for Marin and Sam can now be made with a similar structure. Maybe you are even thinking about another component for the Bio Card!
+
+Once `/services` and the root level HTML files are ported, here is what the finished Gatsby project file structure looks like:
+
+```
+gatsby-site
+├── static
+│   ├── favicon.ico
+│   └── person.png
+├── src
+│   ├── styles
+│   │   ├── normalize.css
+│   │   └── style.css
+│   ├── components
+│   │   └── Layout.js
+│   └── pages
+│       ├── index.js
+│       ├── about.js
+│       ├── contact.js
+│       ├── 404.js
+│       ├── who
+│       │   ├── index.js
+│       │   ├── ellla-arborist.js
+│       │   ├── marin-leafer.js
+│       │   └── sam-surgeon.js
+│       └── services
+│           ├── index.js
+│           ├── growing.js
+│           ├── cleaning.js
+│           └── shrinking.js
+├── node_modules
+├── .gitignore
+├── .prettierignore
+├── .prettierrc
+├── gatsby-config.js
+├── LICENSE
+├── package-lock.json
+├── package.json
+└── README.md
+```
+
+## Building & Deploying
+
+With your new Gatsby application taking shape, it's time to integrate it into your existing HTML website.
+
+### Gatsby build step
+
+You now have a site that mirrors the existing HTML site section. Stop the development server if it's still running; it's time to run the production build! 🎉
+
+```shell
+gatsby build
+```
+
+Once a build is complete, the compiled set of files can be found in `/public`.
+
+### Hosting the new website
+
+Once built, the contents of the `/public` folder are ready to be hosted at the root (`/`) of a domain. The files can be deployed in exactly the same way that your existing HTML site was deployed. For more deployment options, see the [Deploying and Hosting page](/docs/deploying-and-hosting/).
+
+If the Gatsby site is to be hosted at a non-root path, e.g. `example.com/blog/`, Gatsby needs to be informed so page and asset links in the built output can be prefixed.
+
+[Path prefix](/docs/path-prefix/) is the Gatsby option for setting a non-root hosting path. Here is the example project `gatsby-config.js` file with a path prefix added for `/blog`:
+
+```js:title=/gatsby-site/gatsby-config.js
+/**
+ * Configure your Gatsby site with this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/gatsby-config/
+ */
+
+module.exports = {
+  pathPrefix: "/blog", // highlight-line
+  plugins: ["gatsby-plugin-react-helmet"],
+}
+```
+
+If the HTML and non-HTML files of the Gatsby build output are to be served in different locations, this can also be supported with the [Asset Prefix](/docs/asset-prefix/) option.
+
+Once prefix options have been set, the build must be run with the `--prefix-paths` flag to apply them:
+
+```shell
+gatsby build --prefix-paths
+```
+
```

Comment:
What do you think of including a tip about gatsby-serve? I could see this tripping people up otherwise.
````suggestion

You can also serve the built site locally for review using the `prefix-paths` flag:

```shell
gatsby serve --prefix-paths
```
````

Reply:
Excellent! I didn't know about serve 👍